lib/mix/tasks/compile.bundlex.ex

```diff
@@ -23,7 +23,7 @@ defmodule Mix.Tasks.Compile.Bundlex do
     commands = []
 
     app = MixHelper.get_app!()
-    platform = Bundlex.platform()
+    platform = Platform.get_target!()
```

Comment:
Shouldn't we get rid of `Bundlex.Platform` altogether?

Reply:
I think ultimately yes, but that would mean some significant changes